examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<h1>Teil 1: Die grundlegenden Werkzeuge der privaten, dezentralen Datenwissenschaft&nbsp;</h1><p> </p><p>Die erste Frage, die Sie sich vielleicht stellen, lautet: Wie trainieren wir ein Modell an Daten, auf die wir keinen Zugriff haben?&nbsp;</p><p>Die Antwort ist überraschend einfach. Wenn Sie es gewohnt sind, in PyTorch zu arbeiten, sind Sie es gewohnt, mit <code>torch.Tensor</code> Objekten zu arbeiten. Tensor-Objekte wie diese!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>